pyccel/codegen/printing/pycode.py

```diff
@@ -1028,9 +1028,11 @@ def _print_ClassDef(self, expr):
         return classDef
 
     def _print_ConstructorCall(self, expr):
-        cls_name = expr.func.cls_name
-        args = ', '.join(self._print(arg) for arg in expr.arguments)
-        return f"{cls_name}({args})"
+        cls_name = expr.funcdef.cls_name
+        cls_variable = expr.cls_variable
+        if expr.args[1:] is not None:
```

Comment:
I am not sure it is possible for this condition to arise `expr.args[1:]` can be `[]` but it can never be `None`